src/components/Navbar/navbar.scss

```diff
@@ -0,0 +1,121 @@
+@import "../../variables.scss";
+
+.navbar {
+  max-width: 480px;
+  position: relative;
+  .space {
```

Comment:
todo: In CSS, avoid nested classes whenever possible, as it makes hard to do any needed future override. Instead prefer to concat classes names. (eg ass you do below with .navbar-item-btn)

Even better, this should be a `*.modules.scss` file to use CSS Modules and scope these classes, which allows you to keep the class name short without nesting.

Does this make sense to you?

Reply:
until now I thought by using scss you didn't have to worry about scope 😵‍💫 I guess I haven't used css alone as much as styled components or tailwind

Reply:
exactly, SCSS does not solve scope. That's solved by CSS Modules (or css-in-js)